lib/jsdom/living/form-data.js

```diff
@@ -0,0 +1,79 @@
+"use strict";
+
+const formDataSymbols = require("./form-data-symbols");
+
+module.exports = function (core) {
+  const Blob = core.Blob;
+  const File = core.File;
+
+  class FormData {
```

Comment:
That's was my first thought but the specification says that FormData is "an ordered list of entries" and the algorithms described match the list use. (the form data should be sent in this exact order too, even it's not currently the case in my XMLHttpRequest implementation).
https://xhr.spec.whatwg.org/#interface-formdata